lib/commands/command-install

```diff
@@ -95,9 +97,8 @@ install_tool_version() {
     local install_type="version"
 
     if [ "${version_info[0]}" = "latest" ]; then
-      load_cmd "latest"
       local version
-      version=$(latest_command "$plugin_name" "${version_info[1]}")
+      version=$(asdf latest "$plugin_name" "${version_info[1]}")
```

Comment:
Does it make sense to invoke asdf again in this context? It seems like with the command-finding logic this would be slower than just invoking the `latest_command` function directly.